<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewCoursesOfStudent</class>
 <widget class="QWidget" name="ViewCoursesOfStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(5, 14, 43)</string>
  </property>
  <widget class="QLineEdit" name="StudentNamelineEdit_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;
    border-radius: 10px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="StudentIDlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-radius: 10px;
background:white;
color:black;</string>
   </property>
   <property name="text">
    <string>Student ID </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="filterby">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:24px;
color :white;
background-color: transparent;
  font-weight: bold;
</string>
   </property>
   <property name="text">
    <string>Filterd by  :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>831</width>
     <height>300</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>2</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Medium</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AllEditTriggers</set>
   </property>
   <property name="iconSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>143</number>
   </attribute>
   <column>
    <property name="text">
     <string>Grade</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Finished</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Code</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QCheckBox" name="progress">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>40</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
color:white;
font: 16pt &quot;Arial&quot;;
    background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Progress Courses </string>
   </property>
  </widget>
  <widget class="QLabel" name="StudentNamelabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-radius: 10px;
background:white;
color:black;</string>
   </property>
   <property name="text">
    <string>Student Name </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="finished">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
color:white;
font: 16pt &quot;Arial&quot;;
    background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Finished Courses </string>
   </property>
  </widget>
  <widget class="QPushButton" name="view">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">    font-weight: 600;
	font-size:20px;
    border-radius: 20px;
    color: rgb(47, 255, 43);
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #1c335f, stop:1 #13112d);
</string>
   </property>
   <property name="text">
    <string>view</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Student_ID">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;
    border-radius: 10px;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>20000000</number>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="filter">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>150</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">    font-weight: 600;
	font-size:20px;
    border-radius: 20px;
    color: white;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #1c335f, stop:1 #13112d);
</string>
   </property>
   <property name="text">
    <string>filter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>540</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">    font-weight: 600;
	font-size:20px;
    border-radius: 20px;
    color: rgb(255, 33, 25);
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #1c335f, stop:1 #13112d);
</string>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="saved_message">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>540</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;color:black;
    border-radius: 10px;
color : rgb(15, 254, 23) ;</string>
   </property>
   <property name="text">
    <string>Saved Successfully</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
